iOS/ViewControllers/Account/AppearanceSettingsViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AppearanceSettingsViewController: UITableViewController {
+
+    @available(iOS 13.0, *)
+    private var theme: Theme {
+        get {
+            return UserDefaults.standard.theme
+        }
+        set {
+            UserDefaults.standard.theme = newValue
+            self.configureStyle(for: newValue)
+        }
+    }
+
+    @available(iOS 13.0, *)
+    private func configureStyle(for theme: Theme) {
+        UIApplication.shared.windows.forEach { window in
+            window.overrideUserInterfaceStyle = theme.userInterfaceStyle
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 3
+    }
+
+    // delegate
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "SettingsOptionBasicCell", for: indexPath)
+        switch indexPath.row {
+        case 0:
+            cell.textLabel?.text = NSLocalizedString("settings.appearance.device", comment: "Title for selecting device apperance option")
+        case 1:
+            cell.textLabel?.text = NSLocalizedString("settings.appearance.light", comment: "Title for selecting light apperance option")
+        case 2:
+            cell.textLabel?.text = NSLocalizedString("settings.appearance.dark", comment: "Title for selecting dark apperance option")
```

Comment:
After adopting the `CaseIterable` protocol for `Theme`, you can move the title to the `Theme` type (similar to `UserDefaults+VideoQuality.swift`). In this way, you can avoid the `switch` statement and simply use:
```swift
cell.textLabel?.text = Theme.allCases[indexPath.row].title
```